### PR TITLE
Add setting for heapdump folder (issue #45)

### PR DESCRIPTION
**Pull request by andrecrt**
Also change default folder to the app folder (instead of plugin)

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -13,7 +13,8 @@ var Utils = require('./utils');
 var internals = {
     initialized: false,
     defaults: {
-        logPath: Path.join(__dirname, '..', 'poop.log'),
+        logPath: Path.join(Path.dirname(process.mainModule.filename), 'poop.log'),
+        heapdumpFolder: Path.join(Path.dirname(process.mainModule.filename)),
```
Comment (cjihrig) [inline lib/index.js]: I'd prefer to use `process.cwd()` instead. It's simpler and works in the REPL (not that that is a huge deal).

**Comment by cjihrig** [comment]
Closes #45

File: lib/index.js (hunk 1 above)
Comment (andreftavares) [inline lib/index.js]: Do you want me to make that change?

Comment (cjihrig) [inline lib/index.js]: Yes, please.

File: lib/index.js [hunk 2]
```diff
@@ -13,7 +13,8 @@ var Utils = require('./utils');
 var internals = {
     initialized: false,
     defaults: {
-        logPath: Path.join(__dirname, '..', 'poop.log'),
+        logPath: Path.join(process.cwd(), 'poop.log'),
```
Comment (cjihrig) [inline lib/index.js]: Can you please update this in the docs.

File: lib/index.js [hunk 3]
```diff
@@ -13,7 +13,8 @@ var Utils = require('./utils');
...
+        heapdumpFolder: process.cwd(),
```
Comment (cjihrig) [inline lib/index.js]: This needs to be documented.

Comment (cjihrig) [inline lib/index.js]: Actually, I'll take care of these changes.

**Pull request merged**